### ISSUE TITLE
improve handling of album artist in lfmbatchedit.sh

### ISSUE THREAD

**Issue by arcctgx** [arcctgx/lfmedit#3]
This looks like a case when the album field is set, but album artist field is not set on Last.fm side.

```
INF: applyChangesFrom(): editing scrobble 388 of 408
INF: requestScrobbleEdit(): This is the edit that will be applied:
-1291342475	[untitled]	William Basinski	Melancholia	
+1291342475	Melancholia VII	William Basinski	Melancholia	William Basinski
ERR: requestScrobbleEdit(): HTTP error 500: check edit parameters (wrong original album artist?)
```

**Comment by arcctgx** [comment]
No, 3a9f58d doesn't fix it.

**Comment by arcctgx** [comment]
I'm not so sure anymore if it's really a problem with album artist field. I need to reproduce it and debug further.

**Comment by arcctgx** [comment]
I haven't seen this problem in a while.
I think it's finally fixed by aeaac8c.
